include/audacity/PluginInterface.h

```diff
@@ -72,7 +71,6 @@ class AUDACITY_DLL_API PluginManagerInterface /* not final */
 
    virtual const PluginID & RegisterPlugin(ModuleInterface *module) = 0;
    virtual const PluginID & RegisterPlugin(ModuleInterface *provider, EffectDefinitionInterface *effect, int type) = 0;
-   virtual const PluginID & RegisterPlugin(ModuleInterface *provider, ImporterInterface *importer) = 0;
```

Comment:
Its little bit confused me, that means it unnessecary, because for export/import implimentation described like a standalone module?

Reply:
I don't really know the original intention, but the simple fact is that this function was not used.

I think there was an idea of detailed sub-classing of various "Plug-in" types, that turned out to be excessive.